src/audio/copier/copier.c

```diff
@@ -1385,8 +1385,10 @@ static int copier_params(struct comp_dev *dev, struct sof_ipc_stream_params *par
 	struct comp_buffer *sink, *source;
 	struct comp_buffer __sparse_cache *sink_c, *source_c;
 	struct list_item *sink_list;
+	struct comp_dev *child_dev;
```

Comment:
typo in commit title parammeters -> parameters

Reply:
done